Makefile

```diff
@@ -515,6 +516,25 @@ claim_list_height_5: ## List all the claims at height 5
 claim_list_session: ## List all the claims ending at a specific session (specified via SESSION variable)
 	poktrolld --home=$(POKTROLLD_HOME) q supplier list-claims --session-id $(SESSION) --node $(POCKET_NODE)
 
+##############
+### Params ###
+##############
+
+MODULES := application gateway pocket service session supplier tokenomics
+
+# TODO_IMPROVE(#322): Improve once we decide how to handle parameter updates
+.PHONY: update_tokenomics_params
+update_tokenomics_params: ## Update the tokenomics module params
+	poktrolld --home=$(POKTROLLD_HOME) tx tokenomics update-params 43 --keyring-backend test --from pnf --node $(POCKET_NODE)
+
+.PHONY: query_all_params
+query_all_params: ## Query the params from all available modules
+	@for module in $(MODULES); do \
+	    echo "~~~ Querying $$module module params ~~~"; \
+	    poktrolld query $$module params --node $(POCKET_NODE) --output json | jq; \
```

Comment:
> `--output json | jq` :sunglasses: nice!

Reply:
Wdyt about either adding `jq` to `install_ci_deps`, it looks like we're already expecting `npm` to be installed.